dex-load/src/main/scala/com/wavesplatform/dex/load/ws/WsCollectChangesClient.scala

```diff
@@ -37,6 +37,7 @@ class WsCollectChangesClient(apiUri: String, address: String, aus: String, obs:
       }
       orderBookUpdates.put(diff.assetPair, updatedOb)
       none
+    case _ => throw new IllegalArgumentException(s"Unexpected message")
```

Comment:
Let's write the message for easier debugging:
```scala
case x => throw new IllegalArgumentException(s"Unexpected message: $x")
```

Reply:
fixed